### PR TITLE
Use local ort python package in server model tests

### PR DESCRIPTION
**Pull request by NonStatic2014**

### CODE REVIEW

File: cmake/onnx/CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ add_library(onnx_proto ${ONNXRUNTIME_ROOT}/core/protobuf/onnx-ml.proto ${ONNXRUN
 target_include_directories(onnx_proto PUBLIC $<TARGET_PROPERTY:protobuf::libprotobuf,INTERFACE_INCLUDE_DIRECTORIES> "${CMAKE_CURRENT_BINARY_DIR}/..")
 target_compile_definitions(onnx_proto PUBLIC $<TARGET_PROPERTY:protobuf::libprotobuf,INTERFACE_COMPILE_DEFINITIONS>)
 onnxruntime_protobuf_generate(APPEND_PATH IMPORT_DIRS ${ONNXRUNTIME_ROOT}/core/protobuf TARGET onnx_proto)
+onnxruntime_protobuf_generate(APPEND_PATH IMPORT_DIRS ${ONNXRUNTIME_ROOT}/core/protobuf LANGUAGE python TARGET onnx_proto)
```
Comment (pranavsharma) [inline cmake/onnx/CMakeLists.txt]: This is required only when server is built, right?

Comment (NonStatic2014) [reply cmake/onnx/CMakeLists.txt]: Reverted

**Pull request merged**